application-gateway-autoscaling-zone-redundant.md (#33358)
Azure/azure-content: articles/application-gateway/application-gateway-autoscaling-zone-redundant.md

- **Zone redundancy**: An Application Gateway or WAF deployment can span multiple Availability Zones, removing the need to provision separate Application Gateway instances in each zone with a Traffic Manager. You can choose a single zone or multiple zones where Application Gateway instances are deployed, which makes it more resilient to zone failure. The backend pool for applications can be similarly distributed across availability zones.

  Zone redundancy is available only where Azure Zones are available. In other regions, all other features are supported. For more information, see [What are Availability Zones in Azure?](../availability-zones/az-overview.md#services-support-by-region)

> - **Static VIP**: Application gateway v2 SKU supports the static VIP type exclusively. This ensures that the VIP associated with application gateway doesn't change for the lifecycle of the deployment, even after a restart. Since we did not have a static VIP in V1, there was a need to use the App Gateway URL for a domain name routing to the App Services via the Application Gateway, and not the IP address.

Reviewer: ```suggestion
- **Static VIP**: Application Gateway v2 SKU supports the static VIP type exclusively. This ensures that the VIP associated with the application gateway doesn't change for the lifecycle of the deployment, even after a restart.  There isn't a static VIP in v1, so you must use the application gateway URL instead of the IP address for domain name routing to App Services via the application gateway.
```

Reviewer: @neerajks77 
Thanks for your contribution. Please review my changes. Incorporate these and then I'll merge.
